whecho/_config.py

```diff
@@ -7,9 +7,13 @@
 import socket
 import platform
 
+try:
```

Comment:
add comment to this try-except block. Not clear why this exists.

please revise this file

Reply:
getpass.getuser() causes errors when testing on Windows OS. My solution was to hard code "user" as the username for these issues.

Reply:
is there an alternative to `getpass.getuser()` we can look into?

Reply:
comment added. not able to find an alternative at the moment.